src/app/qgisapp.cpp

```diff
@@ -11083,14 +11081,14 @@ void QgisApp::showProgress( int progress, int totalSteps )
   if ( progress == totalSteps )
   {
     mProgressBar->reset();
-    mProgressBar->hide();
+    mProgressBar->setMaximumWidth( 0 );
```

Comment:
I'm not sure about this - I'm concerned it may result in odd artefacts on some platforms/qt themes. Is the maximum height change alone not sufficient to fix this?

Reply:
Darn, your comment popped up just as I clicked on merge.

Reply:
I've checked a couple of qt themes on Linux, no glitches. I used both the max height and the width as double safeguards. 

Let's keep an eye for a visual regression on this in the coming days.